frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.scss

```diff
@@ -0,0 +1,21 @@
+.kubevirt-create-vm-modal__virtual-hardware-tab-container {
+  display: flex;
+  flex-direction: column;
+
+  .virtual-hardware-tab-cd-title {
```

Comment:
I think if we wrote these classes like `.kv-create-vm__virtal-hardware__cd-title` we could avoid nesting